.gitignore

```diff
@@ -9,3 +9,4 @@ __pycache__
 .hypothesis/
 .cache/
 _trial_temp/
+.idea/
```

Comment:
IDE / editor-specific ignores are best placed in your personal (device/OS-specific) global gitignore.

Reply:
ok